src/renderers/SymbolInstanceRenderer.js

```diff
@@ -70,12 +70,7 @@ const extractOverridesHelp = (subLayer: any, output: any) => {
     });
   }
 
-  if (
-    (subLayer._class === 'shapeGroup' ||
-      subLayer._class === 'artboard' ||
-      subLayer._class === 'group') &&
-    subLayer.layers
-  ) {
+  if (subLayer.layers) {
```

Comment:
When I removed this a few weeks ago, I was fairly confident? But I will back this out because I am not 100% sure what my reasoning was.